ebuild-writing/common-mistakes/text.xml

```diff
@@ -13,6 +13,17 @@ writing ebuilds.
 <section>
 <title>Common Ebuild Writing Mistakes</title>
 
+<subsection>
+<title>Unguarded external calls</title>
+<body>
+<p>
+Calls to external tools should be guarded with <c>|| die</c> in almost
+all cases to allow failure to be detected. See
+<uri link="::ebuild-writing/error-handling"/>.
```

Comment:
Is this really a common mistake? We don't even have a tracker bug for it (and AFAICS never had one).

If we are to include it, it should say "`die` or `assert`" I think.

Reply:
It _really_ is. This is easily the most common issue that we have to comment on within proxy-maint@. But I appreciate why you're asking -- we've added a fair amount to this section recently and it's reasonable to be concerned about bloat.

I don't think we've ever bothered with tracker bugs given it's considered a "basic" QA issue and is also somewhat tricky to grep for.